### DISCUSSION TITLE
Is container query something on your radar?

### DISCUSSION THREAD

**Discussion by ekaradon** [mantinedev/mantine#3700] [Feature requests]
Hello,

Thanks for this library, I've a lot of fun using it.
I see that container queries are now supported by emotion and by [all the major browsers](https://caniuse.com/?search=container+query), I was wondering if that's something on your radar and if you have plans, components and utilities for this?

**Comment by rtivital** [comment]
No, it is not planned to introduce utilities for container queries, you can use them in styles
